OpenSR/scripts/server/orders/AttackOrder.as

```diff
@@ -151,14 +274,21 @@ tidy class AttackOrder : Order {
 					fleePos = vec3d();
 			}
 			else {
-				if(!isBound) {
+				// I do not understand why blind mind would want a ship that was ordered
+				// to close in on an enemy ever deciding it should not close in if that
+				// would leave its current region?
+				// perhaps it makes sense when vanilla wouldn't let you tell a ship to close in
+				// manually????
```

Comment:
It also makes sense if you're not allowed to sublight out of your current region. (Flux... such a pain in the neck sometimes.)

Reply:
And yeah, it originally made sense because some of the AutoModes were supposed to keep you bound to a region.

Reply:
The auto modes *do* keep you bound to a region, and I believe flux has always been able to sublight out of a region to move to a target?